GsaGHTests/1_BaseParameters/0_Model/GsaModelTest.cs

```diff
@@ -130,5 +133,39 @@ public void TestModelLengthUnit() {
 
       Assert.Equal(LengthUnit.Foot, UnitMapping.GetUnit(m.ApiModel.UiUnits().LengthLarge));
     }
+
+
+    [Fact]
+    public void AnalysisTaskAndCasesCanBeImportedFromSeedModel() {
+      //seed model to read existing analysis task
+      var seedModel = new GsaModel();
+      seedModel.ApiModel.Open(GsaFile.SteelDesignComplex);
+      List<GsaAnalysisTaskGoo> seedTasks = seedModel.GetAnalysisTasksAndCombinations().Item1;
+      var analysis = new GsaAnalysis();
+      foreach (GsaAnalysisTaskGoo task in seedModel.GetAnalysisTasksAndCombinations().Item1) {
+        analysis.AnalysisTasks.Add(task.Value);
+      }
+      //import into new model
+      var assembly = new ModelAssembly(null, null, null, null, null, null, analysis,
+        LengthUnit.Meter, Length.Zero, false, null);
+      var model = new GsaModel(assembly.GetModel());
+      List<GsaAnalysisTaskGoo> importedTasks = model.GetAnalysisTasksAndCombinations().Item1;
+
```

Comment:
use equality operator on every type so we avoid long loops of asserts

```suggestion
      Assert.Equal(importedTasks, seedTasks);
```

Reply:
I used existing helper class  Assert.True(Duplicates.AreEqual(seedTasks, importedTasks));